app.js

```diff
@@ -113,17 +110,23 @@ app.use(function(req, res, next) {
   next(globalError(404))
 })
 
-// handle global errors.
-app.use(function(err, req, res) {
-  res.locals.message = err.message
-  res.locals.error = req.app.get('env') === 'development' ? err : {}
+// Pass error information to res.locals
+app.use((err, req, res, next) => {
+  let errObj = {}
+
+  let status = err.status || err.statusCode || 500
+  status = status < 400 ? 500 : status
```

Comment:
🤔 doesn't this mean we'll be filtering out certain 400 errors, like 431, which we got before about our header size